frontend/components/zuzalu-purchase.tsx

```diff
@@ -246,7 +244,7 @@ export function ZuzaluPurchaseForm(props: ZuzaluPurchaseFormProps) {
               type: "error",
             });
             return;
-          } else if (!balanceLoading && balance && balance.value.isZero()) {
+          } else if (!balanceLoading && balance && balance.value === 0n) {
```

Comment:
Should we use checkWriteable here? Maybe provide the ability to override the chainID we expect?

For Zuzalu, we are always sending transfers to Ethereum mainnet

Reply:
Fixed!